hasher-matcher-actioner/hmalib/common/label_models.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import hmalib.common.config as config
+import json
+import typing as t
+
+from dataclasses import dataclass, field, fields
+from hmalib.common.logging import get_logger
+from requests import get, post, put, delete, Response
+
+logger = get_logger(__name__)
+
+
+@dataclass(unsafe_hash=True)
+class Label:
+    key: str
+    value: str
+
+    def to_dynamodb_dict(self) -> dict:
```

Comment:
Please check master (view this file in github) and I think you'll see the to_ and from_dynamodb_dict functions were removed in my version of the file.